moveit_core/collision_detection/include/moveit/collision_detection/collision_common.h

```diff
@@ -222,16 +223,22 @@ struct CollisionRequest
 
 namespace DistanceRequestTypes
 {
+/** \brief Type of distance request */
 enum DistanceRequestType
 {
```

Comment:
Personally, I don't like this kind of bloat. Was that really necessary? I guess you should change the comments to:
`GLOBAL,  ///< Find the global minimum`